src/components/TopNavigation/TopNavigation.tsx

```diff
@@ -0,0 +1,37 @@
+import Image from "next/image";
+import ExampleImage from "@/public/images/example-image.png";
+import AlertIcon from "@/public/icons/alert.svg";
+import ShoppingListIcon from "@/public/icons/shopping-list.svg";
+import Link from "next/link";
+
+const TopNavigation = () => {
+  const userName: string = "John Doe";
+  return (
+    <>
+      <div className="relative">
```

Comment:
note for future, use flex or grid layout, absolute positioning doesnt scale. we can also make it sticky